src/index.js

```diff
@@ -19,9 +19,11 @@ function makeShouldSkip () {
   return function shouldSkip (file, opts) {
     if (!exclude) {
       const cwd = getRealpath(process.env.NYC_CWD || process.cwd())
+      const include = process.env.NYC_CONFIG ? JSON.parse(process.env.NYC_CONFIG).include : false
       exclude = testExclude(assign(
         { cwd },
         Object.keys(opts).length > 0 ? opts : {
+          include: include,
```

Comment:
I think I'd prefer that we only set include if `NYC_CONFIG.include` has been found; perhaps pull this object out into an object called `defaults`; and only set the `include` key if it's found?